config/examples/Creality/Ender-3 Pro/CrealityV422/Configuration_adv.h

```diff
@@ -3218,7 +3217,7 @@
    * Beta feature!
    * Create a 50/50 square wave step pulse optimal for stepper drivers.
    */
-  #define SQUARE_WAVE_STEPPING
+  //#define SQUARE_WAVE_STEPPING
```

Comment:
I've not done any testings on this feature to know what type of boards it helps but seems to be disabled in 60% of Ender configs and enabled in 40%.

Reply:
It's commented out in CrealityV427, and it's listed as a beta feature. I've had no problems running it either, so I can certainly re-enable it, but it should probably be consistent between the two configurations.

Reply:
I looked into this one more and its inside an #if TMC block so its value is ignored anyways for V422 boards (since it doesn't use TMC driver even for subset of V422 boards that actually have TMCs).  So it does make sense to comment it out.